spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/ConditionalRejectingErrorHandler.java

```diff
@@ -187,6 +182,22 @@ private boolean isCauseFatal(Throwable cause) {
 					|| isUserCauseFatal(cause);
 		}
 
+		/**
+		 * Log the fatal ListenerExecutionFailedException at WARN level, excluding stack trace.
+		 * Subclasses can override this behavior.
+		 * @param t the {@link ListenerExecutionFailedException}.
+		 * @param cause the cause
+		 * @since 2.2.4
+		 */
+		protected void logFatalException(ListenerExecutionFailedException t, Throwable cause) {
```

Comment:
I added the `cause` parameter here so the subclass can algo use the (already properly determined) cause if they so desire.

Not sure about the method name here, though I think this one is good enough.